cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/CvrSampleListUtil.java

```diff
@@ -54,7 +54,7 @@ public class CvrSampleListUtil {
     private Set<String> newDmpGmlPatients = new HashSet<>();
     private Map<String, List<String>> gmlPatientSampleMap = new HashMap<>();
     private Integer maxNumSamplesToRemove;
-    private Set<String> samplesRemovedList = new HashSet<>();
+    //private Set<String> samplesRemovedList = new HashSet<>();
```

Comment:
I removed `getSamplesRemovedList()` completely.  The list is sent in an email. TODO print to log file that sample has been removed.  Actually this was printed to the log file at the end – maybe it can be done along the way instead?  They were listed in an email as well as output to the log file.  This is the only change to the logfile (besides timings and timestamps etc.).